odoo/addons/base/models/res_users.py

```diff
@@ -338,12 +328,14 @@ def _get_default_image(self):
                                  compute='_compute_accesses_count', compute_sudo=True)
     groups_count = fields.Integer('# Groups', help='Number of groups that apply to the current user',
                                   compute='_compute_accesses_count', compute_sudo=True)
-    image_1920 = fields.Image(related='partner_id.image_1920', inherited=True, readonly=False, default=_get_default_image)
 
     _sql_constraints = [
         ('login_key', 'UNIQUE (login)',  'You can not have two users with the same login !')
     ]
 
+    def _avatar_get_seed(self):
```

Comment:
I don't see any difference from standard get_seed ...

Reply:
`res.users` doesn't have the mixin, it only inherits the fields from `res.partner` (but not the methods)

Reply:
Should probably inherit from the mixin then. Makes no sense to have fields and override all methods, that's the purpose of a mixin

Reply:
`_avatar_get_seed` is only defined in models that inherit from avatar.mixin. res_users doesn't inherit from it, so we need to define the method here

Reply:
I don't think we want to use the mixin on user, we don't have user to have its own image and avatar fields, it should re-use those from its related partner instead.

Reply:
Yeah, you manage avatars on user model, redefine all methods, but you don't need the mixin.

Please add the mixin or find another way to reduce code.

Reply:
Is there a confusion here? Almost nothing is redefined in user, the seed is the only method that is defined on user, and user itself doesn't even need it, it's only to avoid having to duplicate logic of `self.name + str(self.create_date.timestamp())` in both partner and employee.